virttest/qemu_vm.py

```diff
@@ -898,11 +898,17 @@ def add_memorys(devices, params):
             """
             options = []
             mem_params = params.object_params("mem")
-            if mem_params.get("maxmem"):
+            mem_params.setdefault("mem", "512")
+            automem = mem_params.get("automem", "yes") == "yes"
```

Comment:
origin code will reset destination VM memory size if no enough free memory space to  launch destination VM.  if migration failed because different memory size, it's very hard for tester to found it. I think better to make test failed with real reason.